### PR TITLE
Added new format performance

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: content/reference/sql/benchmarks/latency.md [hunk 1]
```diff
@@ -60,4 +62,42 @@ The Dolt version is `0.40.29`.
...
 |-----------------------|-----|
 <!-- END_LATENCY_RESULTS_TABLE -->
+
+### New Format (`__DOLT__`)
+
+Below are the results of running `sysbench` MySQL tests against Dolt
+SQL Server for the most recent release of Dolt in the [new 
+storage format](https://www.dolthub.com/blog/2022-08-12-new-format-migraiton/).
+This is not updated automatically with each release yet.
+To get this performance, craete your database with `dolt init --new-format`.
```
Comment (andy-wm-arthur) [inline content/reference/sql/benchmarks/latency.md]:
```suggestion
To get this performance, create your database with `dolt init --new-format`. 
```

File: content/reference/sql/benchmarks/latency.md [hunk 2]
```diff
@@ -60,4 +62,42 @@ The Dolt version is `0.40.29`.
...
+SQL Server for the most recent release of Dolt in the [new 
+storage format](https://www.dolthub.com/blog/2022-08-12-new-format-migraiton/).
+This is not updated automatically with each release yet.
+To get this performance, craete your database with `dolt init --new-format`. 
+
+|       Read Tests        | MySQL |  Dolt  | Multiple |
+|------------------------ |-------|--------|----------|
+| covering\_index_scan    |   2.0 |  2.81  |      1.5 |
+| groupby\_scan           |  12.3 | 24.83  |      2.0 |
+| index\_join_scan        |  1.12 |  7.98  |      7.0 |
```
Comment (andy-wm-arthur) [inline content/reference/sql/benchmarks/latency.md:79]: there's a new benchmark here called `index_join`

**Review by andy-wm-arthur — APPROVED** [review]

File: content/reference/sql/benchmarks/latency.md [hunk 3]
```diff
@@ -60,4 +62,42 @@ The Dolt version is `0.40.29`.
 | Overall Mean Multiple | 8.2 |
 |-----------------------|-----|
 <!-- END_LATENCY_RESULTS_TABLE -->
+
+### New Format (`__DOLT__`)
+
+Below are the results of running `sysbench` MySQL tests against Dolt
+SQL Server for the most recent release of Dolt in the [new 
+storage format](https://www.dolthub.com/blog/2022-08-12-new-format-migraiton/).
```
Comment (coffeegoddd) [inline content/reference/sql/benchmarks/latency.md:70]:
```suggestion
storage format](https://www.dolthub.com/blog/2022-08-12-new-format-migration/).
```

**Review by coffeegoddd — APPROVED** [review]
I don't think it breaks automated updates... ill find some time to auto update these results too

File: content/reference/sql/benchmarks/latency.md (hunk 3 above)
Comment (andy-wm-arthur) [reply content/reference/sql/benchmarks/latency.md:70]: the link is correct actually.... taking my shame

Comment (timsehn) [reply content/reference/sql/benchmarks/latency.md:70]: Unfortunately, the mispelled one is aktually the link :-)

Try: https://www.dolthub.com/blog/2022-08-12-new-format-migration/

File: content/reference/sql/benchmarks/latency.md (hunk 2 above)
Comment (timsehn) [reply content/reference/sql/benchmarks/latency.md:79]: added

**Pull request merged**